docs/maintenance.md

```diff
@@ -0,0 +1,81 @@
+Maintenance guide
+=================
+
+This is the maintenance guide for pvc-autoresizer.
+
+How to upgrade supported Kubernetes version
+-------------------------------------------
+
+pvc-autoresizer depends on some Kubernetes repositories like `k8s.io/client-go` and should support 3 consecutive Kubernets versions at a time.
+Here is the guide for how to upgrade the supported versions.
+Issues and PRs related to the last upgrade task also help you understand how to upgrade the supported versions,
+so checking them together with this guide is recommended when you do this task.
+
+### Check release notes
+
+First of all, we should have a look at the release notes in the order below.
+
+1. Kubernetes
+    - Choose the next version and check the [release note](https://kubernetes.io/docs/setup/release/notes/). e.g. 1.17, 1.18, 1.19 -> 1.18, 1.19, 1.20
+    - Read the [release note](https://github.com/kubernetes-sigs/controller-runtime/releases), and check which version is compatible with the Kubernetes versions.
+    - Read the [kubebuilder go.mod](https://github.com/kubernetes-sigs/kubebuilder/blob/master/go.mod), and check the controller-tools version corresponding to controller-runtime.
+2. TopoLVM
+    - Choose the [TopoLVM](https://github.com/topolvm/topolvm/releases) version that supported target Kubernetes version.
+3. Depending tools
+    - They does not depend on other software, use latest versions.
```

Comment:
It's OK to use the latest versions of the following tools because these tools don't depend on each other.